test/e2e/tests/onboarding/onboarding.spec.ts

```diff
@@ -251,13 +251,13 @@ describe('MetaMask onboarding', function () {
         await homePage.checkPageIsLoaded();
 
         // Fiat value should be displayed as we mock the price and that is not a 'test network'
-        await homePage.checkExpectedBalanceIsDisplayed('17,000.00', '$');
+        await homePage.checkExpectedBalanceIsDisplayed('10', 'ETH');
 
         // Check for network addition toast
         // Note: With sidepanel enabled, appState is lost during page reload,
         // so the toast notification won't appear. The successful balance display
         // above confirms the network was added correctly.
-        if (await isSidePanelEnabled()) {
+        if (process.env.SELENIUM_BROWSER = 'chrome') {
```

Comment:
this: isSidePanelEnabled is not working, so we need to use the browser

cc @ameliejyc